### PR TITLE
merge Release 2.13 branch into master

### PR DESCRIPTION
**Pull request by krasi-georgiev**
@bwplotka , @simonpasquier @brancz @brian-brazil @juliusv @SuperQ @codesome @csmarchbanks

### CODE REVIEW

File: .promu.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ build:
           path: ./cmd/prometheus
         - name: promtool
           path: ./cmd/promtool
+        - name: tsdb
```
Comment (brian-brazil) [inline .promu.yml]: Does this change make sense? @SuperQ

Comment (SuperQ) [reply .promu.yml]: The change is fine, but I think it's already in master, so I'm not sure why there is a diff here.

Comment (krasi-georgiev) [reply .promu.yml]: I was wondering as well, but it doesn't break anything so didn't spend much time to find out the cause.

**Comment by krasi-georgiev** [comment]
superseded by https://github.com/prometheus/prometheus/pull/6117

**Pull request closed without merging**